browser/extensions/brave_browser_action_util.cc

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+#include "brave/browser/extensions/brave_browser_action_util.h"
+
+#include <memory>
+#include <string>
+
+#include "brave/browser/ui/brave_actions/brave_action_view_controller.h"
+#include "brave/browser/ui/views/brave_actions/brave_actions_container.h"
+#include "brave/browser/ui/views/location_bar/brave_location_bar_view.h"
+#include "brave/browser/ui/views/toolbar/brave_toolbar_view.h"
+#include "chrome/browser/ui/views/frame/browser_view.h"
```

Comment:
As discussed in DM, these views header files should not be included here.

Reply:
Changed, now using an obserer pattern